src/shared/components/palette/utils/first-error.pipe.ts

```diff
@@ -6,24 +6,28 @@ import { ValidationErrors } from '@angular/forms';
 })
 export class FirstErrorPipe implements PipeTransform {
 
-  transform(value: ValidationErrors): string {
+  transform(value: ValidationErrors, args?: string): string {
     if (!value) {
       return '';
     }
 
+    if (!args) {
+      args = 'field';
+    }
+
     let keys = Object.keys(value);
 
     if (!keys.length) {
       return '';
     }
     if (keys[0] ===  'required') {
```

Comment:
This should be a switch statement on `keys[0]` or, better, put that into a `const` beforehand:
```
const firstKey = keys[0];
switch (firstKey) {
  case 'required':
    return `${args} is required`;
  case 'pattern':
    // etc.
}
```

Reply:
Replaced If condition with Switch case